test/SimpleWebRequest.spec.ts

```diff
@@ -80,6 +93,7 @@ describe('SimpleWebRequest', () => {
         expect(
             () => new SimpleWebRequest<string>(url, method, {}, () => headers).start()
         ).toThrowError(`Don't set Accept with options.headers -- use it with the options.acceptType property`)
+        test_resetQueues();
```

Comment:
why is this needed? Shoudln't we clear the queue by executing as a part of the tests?

Reply:
If it throws an error, the queue is not cleared. Added it since my other tests were failing, as they test the queue order :) 
Not sure if this should be addressed in the throwing code itself - WDYT?

Reply:
Oh, that's interesting - not sure what we should do in that case to be honest